### PR TITLE
Updated Card, Input, and Button components + small tweaks

### PR DESCRIPTION
**Pull request by quietbits**
- Updated `Card` component
- Added `isPill` prop to `Button` and `Input`
- Added `isExtraPadding` prop to `Button`
- Small tweaks

### CODE REVIEW

File: @stellar/design-system/src/components/Button/styles.scss [hunk 1]
```diff
@@ -59,7 +59,9 @@
   background-color: var(--Button-color-background-default);
   border-radius: pxToRem(4px);
   cursor: pointer;
-  transition: all var(--anim-transition-default);
+  transition: color var(--anim-transition-default),
```
Comment (aristidesstaffieri) [inline @stellar/design-system/src/components/Button/styles.scss:62]: extra space?

**Review by aristidesstaffieri — APPROVED** [review]
lgtm

File: @stellar/design-system/src/components/Button/styles.scss (hunk 1 above)
Comment (quietbits) [reply @stellar/design-system/src/components/Button/styles.scss:62]: It looks OK in my code editor. Thanks for paying attention to little things like this! :raised_hands: 

image

Comment (aristidesstaffieri) [reply @stellar/design-system/src/components/Button/styles.scss:62]: oh whoops, didn't notice those following lines are also part of the transition.

**Pull request merged**